app/form/v1_0/fields/drug-use/add-drugs.ts

```diff
@@ -12,27 +30,27 @@ const selectMisusedDrugs: FormWizard.Field = {
   multiple: true,
   validate: [{ type: ValidationType.Required, message: 'Select which drugs they have used' }],
   options: [
-    { text: 'Amphetamines (including speed, methamphetamine)', value: 'AMPHETAMINES', kind: 'option' },
-    { text: 'Benzodiazepines (including diazepam, xanex)', value: 'BENZODIAZEPINES', kind: 'option' },
-    { text: 'Cannabis', value: 'CANNABIS', kind: 'option' },
-    { text: 'Cocaine', value: 'COCAINE', kind: 'option' },
-    { text: 'Crack cocaine', value: 'CRACK', kind: 'option' },
-    { text: 'Ecstasy (MDMA)', value: 'ECSTASY', kind: 'option' },
-    { text: 'Hallucinogenics (including ketamine)', value: 'HALLUCINOGENICS', kind: 'option' },
-    { text: 'Heroin', value: 'HEROIN', kind: 'option' },
-    { text: 'Methadone (not prescribed)', value: 'METHADONE_NOT_PRESCRIBED', kind: 'option' },
-    { text: 'Prescribed drugs', value: 'MISUSED_PRESCRIBED_DRUGS', kind: 'option' },
-    { text: 'Other opiates', value: 'OTHER_OPIATES', kind: 'option' },
-    { text: 'Solvents (including gases and glues)', value: 'SOLVENTS', kind: 'option' },
-    { text: 'Steroids', value: 'STEROIDS', kind: 'option' },
-    { text: 'Synthetic cannabinoids (spice)', value: 'SPICE', kind: 'option' },
-    { text: 'Other', value: 'OTHER_DRUG', kind: 'option' },
+    { text: drugsList.AMPHETAMINES.text, value: drugsList.AMPHETAMINES.value, kind: 'option' },
```

Comment:
sorry to be a pita but unless I'm missing a difference, we can simplify this section like the bit below as well.

```options: Object.values(drugsList).map(drug => ({
  text: drug.text,
  value: drug.value,
  kind: 'option',
})),```

Reply:
Resolved in [26618d7](https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/803/commits/26618d774b4fbe835631f009c41087de694e2376)